spqr/spqr/torch_bench.cu

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) SPQR Kernel.2024 Elvir Crncevic (elvircrn@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
What this file for ?

Reply:
I need this for benchmarking, it isn't exposed to spqr, just an internal thing that I need to keep around.